libsentrykube/helm.py

```diff
@@ -46,12 +46,48 @@ def local_path(self, parent: Path) -> Path | None:
         return parent / self.name
 
 
+class HelmReleaseStrategy:
+    @classmethod
+    def from_spec(self, data):
```

Comment:
nit. a pydoc comment with examples of correct data would be great
specifically for dict case since it has some variability.